src/components/DateRangePicker.js

```diff
@@ -269,35 +365,46 @@ class DateRangePicker extends React.Component {
   };
 
   render () {
-    const theme = StyleUtils.mergeTheme(this.props.theme, this.props.primaryColor);
+    const theme = StyleUtils.mergeTheme(
+      this.props.theme,
+      this.props.primaryColor
+    );
     const isLargeOrMediumWindowSize = this._isLargeOrMediumWindowSize(theme);
     const styles = this.styles(theme, isLargeOrMediumWindowSize);
-    const shouldShowCalendarIcon = StyleUtils.getWindowSize(theme.BreakPoints) !== 'small';
+    const shouldShowCalendarIcon =
+      StyleUtils.getWindowSize(theme.BreakPoints) !== 'small';
 
     return (
       <div style={styles.component}>
         <a
           onClick={this._toggleSelectionPane}
-          onKeyUp={(e) => keycode(e) === 'enter' && this._toggleSelectionPane()}
+          onKeyDown={e => keycode(e) === 'enter' && this._toggleSelectionPane()}
           ref={this.props.elementRef}
           style={styles.selectedDateWrapper}
           tabIndex={0}
         >
           {shouldShowCalendarIcon ? (
-            <Icon
-              size={20}
-              style={styles.selectedDateIcon}
-              type='calendar'
-            />
+            <Icon size={20} style={styles.selectedDateIcon} type='calendar' />
           ) : null}
           <div style={styles.selectedDateText}>
+            {/*These below are based on props, not state, as it displays when the date picker is closed - should match the parent*/}
```

Comment:
I don't think this comment is needed.  It reads weird anyway and I'm not even sure what you mean by it.